espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/CreateSession.kt

```diff
@@ -34,9 +34,12 @@ class CreateSession : RequestHandler<SessionParams, Session> {
         try {
             val parsedCaps = parseCapabilities(params.capabilities.firstMatch,
                     params.capabilities.alwaysMatch)
-            val activityName = parsedCaps["appActivity"] as? String
-                    ?: throw InvalidArgumentException("appActivity capability is mandatory")
-            startActivity(parsedCaps["appPackage"] as? String, activityName)
+            val shouldLaunchApp = (parsedCaps["autoLaunch"] ?: true) as Boolean
+            if (shouldLaunchApp) {
+                val activityName = parsedCaps["appActivity"] as? String
```

Comment:
is it safe to ignore this check. Are there any other places in the code that assume this capability must exist?

Reply:
No, there is no other place that checks 'appActivity'